pandas/tests/io/json/test_pandas.py

```diff
@@ -799,107 +798,92 @@ def test_series_non_unique_index(self):
-            )
-            _check_orient(series, "values", dtype=dtype)
-
-            _check_orient(
-                series,
-                "columns",
-                dtype=dtype,
-                numpy=True,
-                check_index_type=check_index_type,
-            )
-            _check_orient(
-                series,
-                "records",
-                dtype=dtype,
-                numpy=True,
-                check_index_type=check_index_type,
-            )
-            _check_orient(
-                series,
-                "split",
-                dtype=dtype,
-                numpy=True,
-                check_index_type=check_index_type,
-            )
-            _check_orient(
-                series,
-                "index",
-                dtype=dtype,
-                numpy=True,
-                check_index_type=check_index_type,
-            )
-            _check_orient(
-                series,
-                "values",
-                dtype=dtype,
-                numpy=True,
-                check_index_type=check_index_type,
-            )
-
-        # basic
-        _check_all_orients(self.series)
+    def test_series_default_orient(self):
         assert self.series.to_json() == self.series.to_json(orient="index")
 
-        objSeries = Series(
+    @pytest.mark.parametrize("numpy", [True, False])
+    def test_series_roundtrip_simple(self, orient, numpy):
+        data = self.series.to_json(orient=orient)
+        result = pd.read_json(data, typ="series", orient=orient, numpy=numpy)
+        expected = self.series.copy()
+
+        if not numpy and PY35 and orient in ("index", "columns"):
+            expected = expected.sort_index()
+        if orient in ("values", "records"):
+            expected = expected.reset_index(drop=True)
+        if orient != "split":
+            expected.name = None
+
+        tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize("dtype", [False, None])
+    @pytest.mark.parametrize("numpy", [True, False])
+    def test_series_roundtrip_object(self, orient, numpy, dtype):
+        # TODO: see why tm.makeObjectSeries provides back DTA
+        dtSeries = Series(
             [str(d) for d in self.objSeries],
             index=self.objSeries.index,
             name=self.objSeries.name,
         )
-        _check_all_orients(objSeries, dtype=False)
+        data = dtSeries.to_json(orient=orient)
+        result = pd.read_json(
+            data, typ="series", orient=orient, numpy=numpy, dtype=dtype
+        )
+        if dtype is False:
+            expected = dtSeries.copy()
+        else:
+            expected = self.objSeries.copy()
+
+        if not numpy and PY35 and orient in ("index", "columns"):
+            expected = expected.sort_index()
+        if orient in ("values", "records"):
+            expected = expected.reset_index(drop=True)
+        if orient != "split":
+            expected.name = None
```

Comment:
L837-L842 looks like it is the same as L810-L815, and really similar to 852-858, 883-884, possibly others.  I guess this is what _check_orient used to be?

If this can be shared, that would be helpful.  If not, comments to the effect of `# this case does foo unlike in test_bar because of baz`